### ISSUE TITLE
What does it meaning by showing 5000/10000 when training?

### ISSUE THREAD

**Issue by helloworldlxb** [fizyr/keras-retinanet#354]
The process of training shows a number over 10000 in console. However, the training set should contain more than 1000000 images. Is the number 10000 only a way to show the progress?
Thank you!

**Comment by hgaiser** [comment]
Yeah maybe we should add more text there :)

It is the evaluation code running on the images. Apparently you have 10000 images in your validation set.

**Comment by helloworldlxb** [comment]
@hgaiser Thank you! But actually it is not validation set. I have 512360 images in the validation set. The number 10000 emerges in the training stage.

**Comment by hgaiser** [comment]
Are you sure? By default the validation set will be used for evaluation (which can be [disabled](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/bin/train.py#L341)). The evaluation prints [these](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/utils/eval.py#L111) messages you referred to.

**Comment by helloworldlxb** [comment]
![tim 20180327155203](https://user-images.githubusercontent.com/12216165/37953970-ee84539c-31d6-11e8-9cdd-e29aa72321b3.png)
@hgaiser Yes. This is the screenshot.

**Comment by hgaiser** [comment]
Ah you mean the other counter. That is the `nr_of_steps_done / total_nr_of_steps` for one epoch. So `500/10000` means it has processed 500 batches and will process 10000 in this epoch. Currently a batch probably has one image, so 500 images would've been processed.

**Comment by helloworldlxb** [comment]
@hgaiser Thanks! But is 10000 the upper bound for the number of epochs can be processed in an epoch?

**Comment by hgaiser** [comment]
https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/bin/train.py#L336-L337
